invenio_app_ils/ui/src/invenio_app_ils/common/urls.js

```diff
@@ -13,6 +13,9 @@ export const BackOfficeURLS = {
   loanDetails: `${BackOfficeBase}/loans/:loanPid`,
   patronDetails: `${BackOfficeBase}/users/:userPid`,
   locationList: `${BackOfficeBase}/locations`,
+  // TODO after editor create link estabilished
+  newLoan: `${BackOfficeBase}/loans/create`,
```

Comment:
can you please change this to something like:
```
loanRequest: `${BackOfficeBase}/circulation/loans/request`,
loanCheckout: `${BackOfficeBase}/circulation/loans/checkout`,
```
and then we need to fix backend `create` -> `checkout`